docs/sources/tempo/configuration/_index.md

```diff
@@ -412,8 +412,8 @@ metrics_generator:
             # Enable traces_target_info metrics
             [enable_target_info: <bool> | default = false]
 
-            # Attribute Key to multiply span metrics
-            [span_multiplier_key: <string> | default = ""]
+            # Attribute Key to multiply metrics
```

Comment:
we should indicate that the attribute name is searched for in both resource and span level attributes